custom_components/meural/__init__.py

```diff
@@ -22,8 +22,14 @@ async def async_setup(hass: HomeAssistant, config: dict):
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up Meural from a config entry."""
+    if "email" not in entry.data:
```

Comment:
Sorry, I suggested the wrong check 😅 
```suggestion
    if "password" not in entry.data:
```